zhaquirks/sonoff/__init__.py

```diff
@@ -1 +1,9 @@
 """Quirks for Sonoff devices."""
+from zigpy.quirks import CustomCluster
+
+SONOFF_CLUSTER_ID = 0xFC57
+
+
+class SonoffManufactureCluster(CustomCluster):
+    name = "Sonoff Manufacturer Specicific"
+    cluster_id = SONOFF_CLUSTER_ID
```

Comment:
This works, of course, but maybe it's also enough to define `SONOFF_CLUSTER_ID = 0xFC57` in the quirk itself, as I'm not sure if that ID the same for all Sonoff devices.

The `CustomCluster` also seems a bit overkill for just using the `.cluster_id` in the end.